cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -3,143 +3,31 @@
    Bureau structure page at /the-bureau/bureau-structure/.
    ========================================================================== */
 
-.org-chart {
 
-    .respond-to-min(@bp-sm-min, {
-        width: 90%;
-        margin: 0 auto;
-    });
+@grid_gutter-width__em: unit(@grid_gutter-width / @base-font-size-px, em);
```

Comment:
Elsewhere this is called `@margin__em`.. we should update it here or in the other spots this is used, for eventual moving of this value to `cf-enhancements`.